corehq/apps/hqadmin/migrations/0018_back_populate_deploy_commit.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 2.2.13 on 2020-07-30 18:59
+
+from django.db import migrations
+from corehq.util.django_migrations import skip_on_fresh_install
+
+
+@skip_on_fresh_install
+def back_populate(apps, schema_editor):
+    HqDeploy = apps.get_model('hqadmin', 'hqdeploy')
+    for deploy in HqDeploy.objects.all():
+        if not deploy.commit:
```

Comment:
```suggestion
        if deploy.diff_url and not deploy.commit:
```
`diff_url` is sometimes `None`